apps/login/package.json

```diff
@@ -12,6 +12,7 @@
     "test:integration:watch": "concurrently --names 'mock,test' --kill-others 'pnpm:mock' 'env-cmd -f ./.env.integration start-server-and-test dev http://localhost:3000 \"pnpm nodemon -e js,jsx,ts,tsx,css,scss --ignore \\\"__test__/**\\\" --exec \\\"pnpm test:integration:run\\\"\"'",
     "test:integration:run": "cypress run --config-file ./cypress/cypress.config.ts --quiet",
     "test:integration:open": "cypress open --config-file ./cypress/cypress.config.ts",
+    "test:integration:runall": "concurrently --names 'mock,test' --kill-others 'pnpm:mock' 'env-cmd -f ./.env.integration start-server-and-test dev http://localhost:3000 \"pnpm nodemon -e js,jsx,ts,tsx,css,scss --ignore \\\"__test__/**\\\" --exec \\\"pnpm test:integration:open\\\"\"'",
```

Comment:
what's the difference to `test:integration:watch`?

Reply:
one uses run, the other open. i've change it to `watch:run` and `watch:open` now 👍